File: pages/login_page.py
from selenium.webdriver.common.by import By
from .base_page import BasePage

class LoginPage(BasePage):
    def go_to_login(self):
        self.driver.get('https://magento.softwaretestingboard.com/customer/account/login/')

    def fill_login_form(self, email, password):
        self.type(By.ID, 'email', email)
        self.type(By.ID, 'pass', password)

    def submit(self):
        self.click(By.ID, 'send2') 

File: cypress/pages/LoginPage.js
class LoginPage {
  visit() {
    return cy.visit('/customer/account/login/');
  }

  fillEmail(email) {
    return cy.get('#email').clear().type(email);
  }

  fillPassword(password) {
    return cy.get('#pass').clear().type(password);
  }

  submit() {
    return cy.get('button[title="Sign In"]').click();
  }
}

module.exports = new LoginPage(); 